src/libraries/System.Drawing.Common/src/System/Drawing/DrawingComWrappers.cs

```diff
@@ -244,35 +236,47 @@ internal struct VtblPtr
         }
 #pragma warning restore CS0649
 
-        internal interface IPicture
+        internal interface IPicture : IDisposable
         {
-            static readonly Guid Guid = new Guid(0x7BF80980, 0xBF32, 0x101A, 0x8B, 0xBB, 0, 0xAA, 0x00, 0x30, 0x0C, 0xAB);
+            static readonly Guid IID = new Guid(0x7BF80980, 0xBF32, 0x101A, 0x8B, 0xBB, 0, 0xAA, 0x00, 0x30, 0x0C, 0xAB);
 
             // NOTE: Only SaveAsFile is invoked. The other methods on IPicture are not necessary
 
-            int SaveAsFile(IntPtr pstm, int fSaveMemCopy, out int pcbSize);
+            int SaveAsFile(IntPtr pstm, int fSaveMemCopy, int* pcbSize);
         }
 
-        private class PictureWrapper : IPicture
+        private unsafe class PictureWrapper : IPicture
         {
             private readonly IntPtr _wrappedInstance;
-            private readonly IPictureVtbl _vtable;
+            private readonly IPictureVtbl* _vtable;
```

Comment:
Nit: It is unusual to cache `_vtable` like this. The vtable to call through is typically fetched at the callsite.

Reply:
An example of how to do this is in `Marshal.QueryInterface()`:

https://github.com/dotnet/runtime/blob/f55390a2e57cfe38fa692032195ed5a9f073e1d9/src/libraries/System.Private.CoreLib/src/System/Runtime/InteropServices/Marshal.cs#L161-L165

I'm not opposed to unpacking the single line if desired.

Reply:
Fixed.